perun-core/src/test/java/cz/metacentrum/perun/core/entry/ResourcesManagerEntryIntegrationTest.java

```diff
@@ -497,6 +497,25 @@ public void getAssignedGroups() throws Exception {
 
 	}
 
+	@Test
+	public void getAssignedGroupsResourceMember() throws Exception {
+		System.out.println(CLASS_NAME + "getAssignedGroupsResourceMember");
+
+		vo = setUpVo();
+		member = setUpMember(vo);
+		group = setUpGroup(vo, member);
+		facility = setUpFacility();
+		resource = setUpResource();
+
+		resourcesManager.assignGroupToResource(sess, group, resource);
+
+		List<Group> groups = resourcesManager.getAssignedGroups(sess, resource, member);
+		groups.forEach(System.out::println);
```

Comment:
We do not print objects in tests, just for debug when writing the code. Test should output it's name and in case of fail, reason for it.